### PR TITLE
Create npm2deb debian bullseye branch

### PR DESCRIPTION
**Pull request by naruxde**
Please take a look at this. The Debian files were created by npm2deb and received a little polish.

Please also test the package that you can build from this branch or find in our OneDrive!

What I find very nice is the line break and the formatting in the control file at e.g. Build-Depends @tboehler1 @iluminat23 - I am almost inclined to propose this as a new standard in our projects 🤣

The changelog contains the complete Git history - It's a first suggestion.

### CODE REVIEW

File: debian/upstream/metadata
Comment (tboehler1) [inline debian/upstream/metadata:7]: Is upstream (e.g. erminas) the correct contact for issues and should the links in `Repository` and `Repository-Browse` lead to them, not to us? If I'm understanding correctly the repository served is this one (RevolutionPi/node-red-...), not the one from erminas.

File: debian/changelog
Comment (tboehler1) [inline debian/changelog]:
```suggestion
node-red-contrib-revpi-nodes (1.1.0-1+revpi11+1) bullseye; urgency=medium
```

File: debian/control
Comment (tboehler1) [inline debian/control]: Is this the correct information, or should this repository be given instead?

File: debian/copyright
Comment (tboehler1) [inline debian/copyright:4]: Same here, is this the correct information?

File: debian/control
Comment (iluminat23) [inline debian/control]:
```suggestion
Vcs-Git: https://github.com/erminas/node-red-contrib-revpi-nodes.git
```

**Review by tboehler1 — CHANGES_REQUESTED** [review]
All in all I'm pleasantly surprised with the result of `npm2deb`, there are just a few things I'm unsure of.

Regarding the format of the `Build-Depends` and `Depends` fields: I don't mind staying with the old format and I don't mind the "new" one, but for the sake of consistency I think we should probably stick with the "old" format where the comma is at the end of a line.

File: debian/upstream/metadata
Comment (iluminat23) [reply debian/upstream/metadata:7]: No, this is correct as it is the upstream metadata not the package metadata.

**Review by iluminat23 — CHANGES_REQUESTED** [review]

**Comment by iluminat23** [comment]
I find the `,` at the begin of the line more disturbing. As it reminds me of the semicolon (`;`) comments in some config files. And I need to remind my self that this is not the case here.

File: debian/rules
Comment (tboehler1) [inline debian/rules:18]: `/usr/lib/node_modules` seems to be the correct path for npm's global `node_modules` directory on the RevPi. The path can be retrieved with `npm root -g`, and this shows a different prefix (`/usr/local/lib/node_modules`) for my laptop.

**Assuming** that Node-RED looks for additional nodes in npm's global `node_modules` directory this path can't be hard coded like it is here and needs to be adjusted. Additionally, Debian specifies directories where node modules should live here: https://wiki.debian.org/Javascript/Nodejs#Packaging_modules
While testing these directories Node-RED didn't pick up the nodes, though.

**Review by tboehler1 — APPROVED** [review]
I wasn't able to find documentation for Node-RED on where it looks for nodes exactly, so testing manually is all I had. The only working method for installing nodes was in the global npm root (`npm root -g`).
Do keep in mind, though, that the global npm root is not the directory that Debian usually installs modules.

Not a blocker, just more information. The directories are fine by me, but do wait for the review from @iluminat23.

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**
